File: js/programmers/level_0/모음 제거.js
// Lv. 0 모음 제거

/**
 * @param {string} my_string
 * @returns {string}
 */

const solution = (my_string) => {
    const arr = [];

    for (const char of my_string) {
        if (
            char !== "a" &&
            char !== "e" &&
            char !== "o" &&
            char !== "i" &&
            char !== "u"
        )
            arr.push(char);
    }

    return arr.join("");
};

File: java/programmers/level_0/src/모음_제거.java
public class 모음_제거 {
//영어에선 a, e, i, o, u 다섯 가지 알파벳을 모음으로 분류합니다.
//문자열 my_string이 매개변수로 주어질 때 모음을 제거한 문자열을 return하도록 solution 함수를 완성해주세요.

  public static String solution(String my_string) {
    String answer = "";

    for (int i = 0; i < my_string.length(); i++) {
      char ch = my_string.charAt(i);
      if (ch != 'a' && ch != 'e' && ch != 'i' && ch != 'o' && ch != 'u') {
        answer += ch;
      }
    }

    return answer;
  }

  public static void main(String[] args) {
    System.out.println(solution("Hello my name is String"));
  }
}